server/.env.example

```
@@ -4,4 +4,6 @@ JWT_SECRET=
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 RESEND_API_KEY=
-
+GITHUB_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_CLIENT_ID key should go before the GOOGLE_CLIENT_ID key

(UnorderedKey)